src/wizards/tokenSwapDealWizard/stages/leadDetailsStage/leadDetailsStage.html

```diff
@@ -80,7 +80,7 @@ <h2 class="heading heading3 title">Make Offers Private?</h2>
         <pform-input
           class="wizardFormInput"
           label="Make Offers Private">
-          <ptoggle value.bind="wizardState.registrationData.offersPrivate"></ptoggle>
+          <ptoggle value.bind="wizardState.registrationData.offersPrivate || false"></ptoggle>
```

Comment:
this should not be necessary.  Remove this and make sure offersPrivate  is initialized to false when the data is created for the first time.

Reply:
This is the place where `offersPrivate` is defaulted to `false` -> enough?
https://github.com/PrimeDAO/prime-deals-dapp/blob/59453eb48011d502b6243ab5c956742927fe12ec/src/entities/DealRegistrationTokenSwap.ts#L133